src/blog-utils/StaticAddonCard/index.js

```diff
@@ -73,15 +73,11 @@ export const StaticAddonCardBase = ({
       </div>
 
       <div className="StaticAddonCard-metadata">
-        {averageRating && (
-          <Rating rating={averageRating} readOnly styleSize="small" />
-        )}
-
-        {averageDailyUsers && (
-          <p className="StaticAddonCard-metadata-adu">
-            Users: {i18n.formatNumber(averageDailyUsers)}
-          </p>
-        )}
+        <Rating rating={averageRating} readOnly styleSize="small" />
+
+        <p className="StaticAddonCard-metadata-adu">
+          Users: {i18n.formatNumber(averageDailyUsers)}
+        </p>
```

Comment:
I guess this is an acceptable fix, but I'd be interested in knowing @bobsilverberg 's opinion on doing this (i.e. always showing the UI, even when 0) rather than correcting the style layout bug when ratings.average or adu is 0.

Reply:
I guess this requirement came from the fact that an add-on with no reviews looks like it has no stars, i.e., a very poor rating, whereas it really just doesn't have a rating.

It does seem odd that we'd feature an add-on with zero ratings in a blog post, so I don't really think this is an issue. Of course, we have now seen an example of that happening!

I suppose it might be better to keep the logic that hides the rating if there are no reviews, for the reason I stated above, but I am also fine to accept this patch as is.